libraries/Espressif.kicad_sym

```diff
@@ -3408,7 +3423,7 @@
     (property "ki_keywords" "ESP32" (id 4) (at 0 0 0)
       (effects (font (size 1.27 1.27)) hide)
     )
-    (property "ki_description" "ESP32-WROVER-E and ESP32-WROVER-IE are two powerful, generic WiFi-BT-BLE MCU modules that target a wide variety of applications, ranging from low-power sensor networks to the most demanding tasks, such as voice encoding, music streaming and MP3 decoding. ESP32-WROVER-E comes with a PCB antenna, and ESP32-WROVER-IE with an IPEX antenna. They both featurea 4 MB external SPI flash and an additional 8 MB SPI Pseudo static RAM (PSRAM)." (id 5) (at 0 0 0)
+    (property "ki_description" "2.4 GHz WiFi + Bluetooth ® + Bluetooth LE module Built around ESP32 series of SoCs, Xtensa® dualcore 32bit LX6 microprocessor 4/8/16 MB flash available 26 GPIOs, rich set of peripherals Onboard PCB antenna or external antenna connector" (id 5) (at 0 0 0)
```

Comment:
PTAL @ESP-Marius

Reply:
Sounds good to me, thanks for the quick fix!

Reply:
Or we can just add the same description as found on the Espressif website for a shorter version.

`ESP32-WROOM-32E integrates ESP32-D0WD-V3, with higher stability and safety performance.`

Reply:
Sure, I dont think the description here matters too much.